components/desktop/nimbus/nimbus.p5m

```diff
@@ -22,6 +23,8 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
+depend type=require fmri=gnome/theme/gnome-icon-theme
```

Comment:
Do we know which icons are missing?

Reply:
I've already added to nimbus some missing icons, for example, https://github.com/OpenIndiana/nimbus/commit/da64fb0aecec84ec48ee4f69c05e0836a6e63a61 . Perhaps, we have just add several more icons.

Reply:
I dumped the runtime require, I'll have a look for what icons are missing in Nimbus. If we are not confident that we found every icon missing, we can always add the require.